.gitignore

```diff
@@ -26,3 +26,9 @@ release_deps
 collector*.yaml
 manager*.yaml
 logging*.yaml
+
+#m365recv dev files
+m365metricsconfig.yaml
```

Comment:
If these just live in your local or tmp directory, there's no need to add them specifically to the gitignore.

Reply:
gotcha